pkg/importer/http-datasource.go

```diff
@@ -204,58 +209,54 @@ func (hs *HTTPDataSource) Close() error {
 	return err
 }
 
-func createHTTPClient(certDir string) (*http.Client, error) {
-	client := &http.Client{
-		// Don't set timeout here, since that will be an absolute timeout, we need a relative to last progress timeout.
-	}
-
-	if certDir == "" {
-		return client, nil
-	}
+// CURL either supports a hashed-directory of certificates or a single file certificate.
+// Create a single file containing all our certificates.
+func createCAFile(certDir string) error {
+	systemCertDir := "/etc/pki/tls/certs/"
+	certDirs := []string{certDir, systemCertDir, common.ImporterProxyCertDir}
 
-	// let's get system certs as well
-	certPool, err := x509.SystemCertPool()
+	certFile, err := os.Create(common.ImporterCertFile)
 	if err != nil {
-		return nil, errors.Wrap(err, "Error getting system certs")
+		klog.Errorf("Failed to create %s, got %v", common.ImporterCertFile, err)
 	}
+	defer certFile.Close()
 
-	// append the user-provided trusted CA certificates bundle when making egress connections using proxy
-	if files, err := ioutil.ReadDir(common.ImporterProxyCertDir); err == nil {
+	for dir := range certDirs {
+		files, err := ioutil.ReadDir(certDir)
```

Comment:
Why are looping over the different dirs, and then only reading certDir, I am assuming you want
```go
files, err := ioutils.ReadDir(dir)
```